src/server/routes/conversions.js

```diff
@@ -121,7 +122,7 @@ router.post('/addConversion', async (req, res) => {
 	const validationResult = validate(req.body, validConversion);
 	if (!validationResult.valid) {
 		log.error(`Invalid input for ConversionsAPI. ${validationResult.error}`);
-		res.sendStatus(400);
+        failure(res, 400, 'Invalid input: ' + validationResult.error.toString());
```

Comment:
Could this one say it is for adding a converson (and the log message is, again, not the best and I approved it :-)